admin-api-server/src/analytics/service/analytics.service.ts

```diff
@@ -117,6 +119,39 @@ export class AnalyticsService {
     });
   }
 
+  async getUsers(params: PaginationParams) {
+    const qryRes = await this.storeRepl.query(
+      `SELECT
```

Comment:
Could you change this formatting of the query such that it follows the existing queries style in this repo? In this case here it should be:
```
      `
SELECT
  id,
  address,
  email,
  consent,
  created_at
FROM marketing 
ORDER BY "${params.orderBy}" ${params.orderDirection}
OFFSET ${params.
LIMIT ${params.pageSize}`,
```

Reply:
Done